### DISCUSSION TITLE
Amiibo Chests Don't Spawn

### DISCUSSION THREAD

**Discussion by xCyni** [jordanbtucker/totk-no-loot-boxes#8] [Q&A] [answered]
As stated above, the chests don't spawn and no jingle bells noise is played. I'm playing on Ryujinx 1.1.804, TOTK version 1.1.0, and none of the versions of the mods spawn chests. The extra items, however, do spawn.

**Comment by jordanbtucker** [comment]
- What version of the mod are you using? v1.0.0, v1.1.0, v1.2.0?
- Which amiibo have you tested?
- Have you completed the Great Sky Island and reached Hyrule yet?
- Have you visited the Depths yet?
- The mod only has one file. Is it in this location? `mods\contents\0100F2C0115B6000\NoLootBoxes\romfs\Pack\ResidentCommon.pack.zs`
- Are you willing to share the Ryujinx log files?

**Reply by xCyni** [reply]
I am on v1.1.0.
The amiibos that DO NOT work on v1.1.0 are Bokoblin, Archer Link, Rider Link, TOTK Link, and Loftwing Zelda. All of the SSBU amiibos work fine.
Yes I have.
Yes I have.
Yes it is.
Yes, it is attached.
[Ryujinx_1.1.804_2023-05-18_16-03-00.log](https://github.com/jordanbtucker/totk-no-loot-boxes/files/11512250/Ryujinx_1.1.804_2023-05-18_16-03-00.log)

**Reply by jordanbtucker** [reply]
Looks like you have other mods installed too. Try uninstalling all mods except for No Loot Boxes and see if that fixes it. If so, let me know what the other mods are so I can check for compatibility issues.

**Reply by xCyni** [reply]
Taking all of the mods out have no change. I also have Reshade installed but taking it out made no difference as well.

**Reply by xCyni** [reply]
I can send my Ryujinx folder for you to tinker with if you'd like.

**Reply by xCyni** [reply]
Just scanned BOTW Zelda's amiibo twice. First time the chest spawned but it took 2-3 seconds longer than it should have. The second time the chest did not spawn.

**Reply by xCyni** [reply, marked as answer]
I think I may have figured out the problem. I just scanned Daruk's amiibo three times. The first and second time I got the fabric then the helm but the third time there was no chest. Maybe some of the amiibos that I'm not able to scan I have already recieved the reward for?

Edit: Turns out I was right on that last part. I have already received the rewards from the amiibos that don't have chests spawning. Maybe you can make an alert that says you have already received the rewards from this amiibo.

Thank you.

**Reply by veronica322** [reply]
hola aca alguien conoce a daruk ?

**Comment by jordanbtucker** [comment]
I hope you don't mind if I keep this discussion open. I just makes it more visible if someone else runs into this issue.

**Reply by xCyni** [reply]
Of course, I didn't know that closing the discussion would make it invisible.

**Reply by jordanbtucker** [reply]
No worries. The default view is to only show open discussions, so closing it hides it by default.

**Reply by jordanbtucker** [reply]
And you can always unsubscribe from the discussion if you don't want to receive any notifications on new comments.
